.github/scripts/check-release.sh

```diff
@@ -1,24 +1,31 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -eu -o pipefail
 
-# check_tag $current_tag $file_tag $file_name
-function check_tag {
-  if [[ "$1" != "$2" ]]; then
-      echo "Error: the current tag does not match the version in Cargo.toml: found $2 - expected $1"
```

Comment:
This function also works with `Cargo.lock`. The original error message would be misleading in this case.